config.json

```diff
@@ -0,0 +1,6 @@
+{
+	"supportedLocales": ["en"],
+	"dest": "src/locales",
```

Comment:
right now it only supports one format at the time, so xml maybe we should build that from webmaker-android directly otherwise we will have to teach this library to do multiple at once.